scripts/artifacts-building/remove-container-aio-config.sh

```diff
@@ -18,7 +18,8 @@
 # Remove the env.d configurations that set the build to use
 # container artifacts. We don't want this because container
 # artifacts do not currently exist when this script is used.
-sed -i.bak '/lxc_container_variant: /d' /opt/rpc-openstack/group_vars/*.yml
+sed -i.bak '/lxc_container_variant: /d' /opt/rpc-openstack/group_vars/*.yml \
```

Comment:
I suspect that the plugin may be loading *any* files from ``/etc/openstack_deploy/group_vars``, including the .bak files which are being copied over:

```
++ rsync -av --delete /opt/rpc-openstack/group_vars /etc/openstack_deploy/
sending incremental file list
group_vars/
group_vars/cinder_all.yml
group_vars/cinder_all.yml.bak
group_vars/glance_all.yml
group_vars/glance_all.yml.bak
group_vars/heat_all.yml
group_vars/heat_all.yml.bak
group_vars/horizon_all.yml
group_vars/horizon_all.yml.bak
group_vars/ironic_all.yml
group_vars/ironic_all.yml.bak
group_vars/keystone_all.yml
group_vars/keystone_all.yml.bak
group_vars/kibana_all.yml
group_vars/kibana_all.yml.bak
group_vars/log_hosts.yml
group_vars/log_hosts.yml.bak
group_vars/logstash_all.yml
group_vars/logstash_all.yml.bak
group_vars/memcached_container.yml
group_vars/memcached_container.yml.bak
group_vars/neutron_all.yml
group_vars/neutron_all.yml.bak
group_vars/nova_all.yml
group_vars/nova_all.yml.bak
group_vars/repo_all.yml
group_vars/repo_all.yml.bak
group_vars/rsyslog_all.yml
group_vars/rsyslog_all.yml.bak
group_vars/swift_all.yml
group_vars/swift_all.yml.bak
group_vars/utility_all.yml
group_vars/utility_all.yml.bak
group_vars/all/
group_vars/all/apt.yml
group_vars/all/apt.yml.bak
group_vars/all/ceph.yml
group_vars/all/ceph.yml.bak
group_vars/all/elasticsearch.yml
group_vars/all/elasticsearch.yml.bak
group_vars/all/lxc.yml.bak
group_vars/all/osa.yml
group_vars/all/osa.yml.bak
group_vars/all/release.yml
group_vars/all/release.yml.bak
group_vars/all/rpc-o.yml
group_vars/all/rpc-o.yml.bak
```

Maybe we should try ditching the backup when doing the inline editing. It's not necessary and was simply a force of habit.